pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2919,6 +2919,9 @@ type CephBlockPoolRadosNamespaceList struct {
 
 // CephBlockPoolRadosNamespaceSpec represents the specification of a CephBlockPool Rados Namespace
 type CephBlockPoolRadosNamespaceSpec struct {
+	// The name of the CephBlockPoolRadosNamespaceSpec namespace. If not set, the default is the name of the CR.
+	// +optional
+	Name string `json:"radosNamespaceName,omitempty"`
```

Comment:
We need the `name` to match in the yaml
```suggestion
	Name string `json:"name,omitempty"`
```

Reply:
Updated